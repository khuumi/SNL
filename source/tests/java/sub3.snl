start infloatsub:
    do show to 45.53 - 45
done
