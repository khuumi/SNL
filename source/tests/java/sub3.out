0.5300000000000011
